cli/tools/test.rs

```diff
@@ -600,7 +624,9 @@ async fn test_specifiers(
     .buffer_unordered(concurrent_jobs.get())
     .collect::<Vec<Result<Result<(), AnyError>, tokio::task::JoinError>>>();
 
-  let mut reporter = create_reporter(concurrent_jobs.get() > 1);
+  let mut reporter =
+    create_reporter(concurrent_jobs.get() > 1, log_level != Some(Level::Error));
```

Comment:
`log_level != Some(Level::Error)` is true when `--quiet` flag is passed?

Reply:
It's inverted here, the flag is echo rather than mute.
Quiet sets log level to error (https://github.com/denoland/deno/blob/main/cli/flags.rs#L400)